apps/css-workshop/pages/assets/avatar.js

```diff
@@ -35,9 +35,10 @@ class Avatar extends HTMLElement {
 
     const innerHtml = `
       <span
-        class="iui-avatar iui-${size}"
+        class="iui-avatar"
         title="${person.title}"
         style="background-color: ${person.color};"
+        ${size ? 'data-iui-size=' + size : ''}
```

Comment:
Should `size` be in double quotes? It seems to work without it also, but having the double quotes would be safer.

```suggestion
        ${size ? `data-iui-size="${size}"` : ''}
```

Same for the `status` on the next line.

Reply:
Done in this [commit.](https://github.com/iTwin/iTwinUI/pull/1935/commits/62c831689c5e557abaa65b7cbd500041c534045f)